frontend/src/Editor/Components/components.js

```diff
@@ -1491,7 +1491,7 @@ export const componentTypes = [
       visibility: { type: 'toggle', displayName: 'Visibility' },
       disabledState: { type: 'toggle', displayName: 'Disable' },
     },
-    exposedVariables: {},
+    exposedVariables: { defaultTab: '' },
```

Comment:
`defaultTab` is the initial tab. I don't think e should change it when the tab is changed.

Reply:
updated @Navaneeth-pk